Scripts/script-ParseEmailFiles.yml

```diff
@@ -557,7 +568,7 @@ script: |-
               'Subject': subj,
               'HTML': html,
               'Text': text,
-              'Headers': headers,
+              'Headers': str(header_dict.items()),
```

Comment:
removed the "str"
store it as  list of `{'name': header.name, 'value': header.vale}`